src/pages/Payouts/index.tsx

```diff
@@ -68,12 +68,14 @@ export const Payouts = (props: PageProps) => {
               <OpenAssistantIcon page="payouts" title="Payout History" />
             </h4>
             <h2>
-              {payouts.length ? (
+              {payoutsList.length ? (
```

Comment:
Why do you use `payoutsList` instead of `payouts`?
Why did order change?

We want to minimise changes in the original code. Let's understand do we really need to introduce them.

Reply:
1. `payouts` shows all data but `payoutsList` only 60 last days with no-amount values. 
Slicing of last `MAX_PAYOUT_DAYS` is here:
```ts
 // take non-zero rewards in most-recent order
  let payoutsList: AnySubscan = [
    ...payouts.concat(poolClaims).filter((p: AnySubscan) => p.amount > 0),
  ].slice(0, MAX_PAYOUT_DAYS);
  ```
2. We change the order because `payoutsList` in the codebase change the order of the list.  (_re-order rewards based on block timestamp_)
```ts
// re-order rewards based on block timestamp
  payoutsList = payoutsList.sort((a: AnySubscan, b: AnySubscan) => {
    const x = new BN(a.block_timestamp);
    const y = new BN(b.block_timestamp);
    return y.sub(x);
  });
  ```

Reply:
@shamilkhan 
We should not change original code without the reason. In your answer I did not find the reason why should we start using `payoutsList` instead of `payouts`. Please, clarify.

Reply:
You receive `payouts` from Cere Stats. Should it be the full account history by interface that was used previously in Subscan implementation?

Besides that your logic looks correct.

Reply:
In my opinion, yes:
1. By default, data received from the Stats App is not sorted. If you fetch N rows, it does not necessarily mean that you will receive the latest N records for rewards.
2. We need to sort non-zero values. The data is presented in a `"accountName","amount"` format, so we need to use regex on the client side to extract this value and sort.

Reply:
Let me rephrase. Does Subscan implementation return the whole history of `payouts` for some account?

Reply:
Subscan has a property called `raw` which determines the number of rows returned. Currently, in the codebase, this number is set to `100`. 

```ts
 const results = await Promise.all([
        handleFetch(activeAccount, 0, API_ENDPOINTS.subscanPoolRewards),
        handleFetch(activeAccount, 1, API_ENDPOINTS.subscanPoolRewards),
      ]);
     
     const handleFetch = async (
    address: string,
    page: number,
    endpoint: string
  ): Promise<AnySubscan> => {
    const res: Response = await fetch(network.subscanEndpoint + endpoint, {
      headers: {
        'Content-Type': 'application/json',
        'X-API-Key': API_SUBSCAN_KEY,
      },
      body: JSON.stringify({
        row: 100,
        page,
        address,
      }),
      method: 'POST',
    });
    const resJson: AnySubscan = await res.json();
    return resJson;
  };
```